packages/Modal/stories/Modal.stories.js

```diff
@@ -12,6 +13,9 @@ import * as types from "../src/types";
 
 import Modal from "../src";
 
+// eslint-disable-next-line import/no-named-as-default-member
+const { Header, Content, Footer, FocusLock } = Modal;
```

Comment:
Can you not do this?  Consumers will use Paprika stories as their guide and I think it makes it harder to find, for example, any `Modal.Header`s in the code, as there are lots of `Header`s out there.

Reply:
Perhaps for a story it is ok? I tend to prefer the shorthand syntax, but don't mind. Have we established a pattern as in FormElement stories we use this approach